modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/edit_cp_definition_diagram_setting.jsp

```diff
@@ -118,8 +106,39 @@ CPDefinitionDiagramType cpDefinitionDiagramType = cpDefinitionDiagramSettingDisp
 				/>
 			</commerce-ui:panel>
 		</div>
-	</div>
-</aui:form>
+
+		<div class="col-8">
+		<commerce-ui:panel
+			title='<%= LanguageUtil.get(resourceBundle, "diagram-mapping") %>'
+		>
+
+				<%
+				cpDefinitionDiagramType.render(cpDefinitionDiagramSetting, request, PipingServletResponseFactory.createPipingServletResponse(pageContext));
+				%>
+
+			</commerce-ui:panel>
+		</div>
+
+		<div class="col-4">
+
+		<commerce-ui:panel
```

Comment:
I assume `<commerce-ui:panel>` is a child of the `div`, so it doesn't need the empty line, and needs to be indented.

Reply:
sorry was an error commit